ConwaysGameOfLife.Tests/ValidatorShould.cs

```diff
@@ -38,7 +39,7 @@ public void NotAcceptUniverse_WhenInputIsInvalid(string input)
         [Theory]
         [InlineData("0,0", 3, 3)]
         [InlineData("2,1", 3, 3)]

```

Comment:
Can we expand tests on the grid width and length? You've already done the hard work - just add a couple more cases with different dimension grids.